Infrastructure.DataAccess/Migrations/202206140930041_AddRepurchaseInitiatedToItContract.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Infrastructure.DataAccess.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class AddRepurchaseInitiatedToItContract : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.ItContract", "RepurchaseInitiated", c => c.Int());
```

Comment:
You must update the ItContractMap to add an index to this field since it must be exposed in the overview and thereby be filterable